libnymea/interfaces/v2xeenergymeter.json

```diff
@@ -0,0 +1,30 @@
+{
+    "description": "v2xe Energy meters which shows current price and forecast price ",
```

Comment:
Since these are very product specific states, there is no need to have a dedicated interface. You can define the states in the integration plugin as you like, but an interface should only be needed if you have something which is the same for many different things in order to unify them or to have a defined interface for interacting with them.